engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -125,6 +126,8 @@
 	 */
 	private final List<ValidatedValueUnwrapper<?>> validatedValueHandlers;
 
+	private final ClassLoader userClassLoader;
```

Comment:
> Somehow I always think of 'end user' in this case.

Our users are application developers :)

> What's about providedClassLoader.

No big fan, isn't everything provided what is passed to the config? Maybe `externalClassLoader` or `userResourceClassLoader`? I think I still like `userClassLoader` best.